plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -402,4 +402,16 @@ String columnSchema()
             return "%s %s NOT NULL%s".formatted(name, type, propertiesSchema.isEmpty() ? "" : " WITH (%s)".formatted(propertiesSchema));
         }
     }
+
+    @Test
+    void testRenameTable()
+    {
+        assertUpdate("CREATE TABLE faker.default.original_table (id INTEGER, name VARCHAR)");
+        assertUpdate("ALTER TABLE faker.default.original_table RENAME TO faker.default.renamed_table");
+        // original_table should not exist anymore after renaming.
+        assertQueryFails("DESC faker.default.original_table", "line 1:1: Table 'faker.default.original_table' does not exist");
+        // should not allow renaming to an already existing table.
+        assertQueryFails("ALTER TABLE faker.default.renamed_table RENAME TO faker.default.renamed_table", "line 1:1: Target table 'faker.default.renamed_table' already exists");
+        assertUpdate("DROP TABLE faker.default.renamed_table");
+    }
```

Comment:
Please add another test (`testRenameTableAcrossSchema`) renaming it to a different schema.